platform-tooling-support-tests/platform-tooling-support-tests.gradle.kts

```diff
@@ -192,12 +192,6 @@ val test by testing.suites.getting(JvmTestSuite::class) {
 				jvmArgumentProviders += JarPath(project, antJarsClasspath.get(), "antJars")
 				jvmArgumentProviders += MavenDistribution(project, unzipMavenDistribution, mavenDistributionDir)
 
-				if (buildParameters.javaToolchain.version.getOrElse(21) < 24) {
```

Comment:
_Who tests the testers?!_ 😉

Reply:
This was a leftover from before 52fdf592f2b7e420a54290dbe6825cbfb7c78c6e where I moved the `ArchUnit` tests to their own source set and test task.